There's no network in Nix's build environment, so this test always fails.

--- a/tests/io/CMakeLists.txt
+++ b/tests/io/CMakeLists.txt
@@ -13,11 +13,6 @@ ecbuild_add_test( TARGET   eckit_test_filepool
                   SOURCES  test_filepool.cc
                   LIBS     eckit )
 
-ecbuild_add_test( TARGET  eckit_test_urlhandle
-                  SOURCES test_urlhandle.cc
-                  CONDITION HAVE_EXTRA_TESTS AND eckit_HAVE_CURL
-                  LIBS    eckit )
-
 ecbuild_add_test( TARGET  eckit_test_circularbuffer
                   SOURCES test_circularbuffer.cc
                   LIBS    eckit )
